website/api/controllers/docs/view-basic-documentation.js

```diff
@@ -55,6 +55,12 @@ module.exports = {
       }
     }
 
+    let showSwagForm = false;
+    // Check the requesting user's cf-ipcountry to see if they're in the US, and their cf-iplongitude header to see if they're in the contiguous US.
+    if(this.req.headers['cf-ipcountry'] === 'US' && this.req.headers['cf-iplongitude'] > -125) {
```

Comment:
Please use [`req.get()`](https://sailsjs.com/documentation/reference/request-req/req-get) for these instead of `req.headers` (While req.headers is still supported in Sails, it's been deprecated in Express, so it's a good idea use req.get() instead.)

Reply:
Also, can you add a quick explanation of why here?  e.g. just adding "Due to shipping costs, " at the beginning would do the trick

Reply:
Makes sense! I've updated this PR with those changes.